README.md

```diff
@@ -1,7 +1,24 @@
 # Polis
 pol.is an AI powered sentiment gathering platform. More organic than surveys, less effort than focus groups.
 
-## Development
+## 🙋🏾‍♀️ Get Involved
+
+1. Say hi in our **chat** [:speech_balloon:][chat] [`gitter.com/pol-is/polisDeployment`][chat]
+2. Join one of our weekly **open calls** :microphone:
+    - Please please please... Newcomers welcome! [Learn more...][calls-about]
+3. Visit our [**issue tracker**][issues] [:white_check_mark:][issues] to offer your skills & energies
+    - We also keep a [project kanban board][board] [:checkered_flag:][board]
+    - :ear: Pssssst! [Learn how...][contributing] (labels, etc.)
+
+   [chat]: https://gitter.im/pol-is/polisDeployment
+   [calls-about]: /CONTRIBUTING.md#telephone_receiver-open-calls
+   [issues]: https://github.com/pol-is/polisServer/issues
+   [board]: https://github.com/orgs/pol-is/projects/1
+   [contributing]: /CONTRIBUTING.md#how-we-work
+
+<div align="right"><sup>Inspired by <a href="https://tomesh.net/get-involved/">Toronto Mesh content</a>.</sup></div>
```

Comment:
This callout isn't necessary, please remove, other than that looks great!

Reply:
Woo! Sounds good for now!

Having said that, there's a larger conversation about how attribution fosters a culture of gratitude, so I'd like to re-open convo later and find a way to include things like this. The exact format isn't important, but gratitude admittedly feels important to me and how I try to organize :)

Re-ticketed here: https://github.com/pol-is/polisServer/issues/262